apps/backend/src/modules/projects/project.repository.ts

```diff
@@ -35,8 +35,11 @@ class ProjectRepository implements Repository {
 		return item ? ProjectEntity.initialize(item) : null;
 	}
 
-	public async findAll(): Promise<ProjectEntity[]> {
-		const projects = await this.projectModel.query().execute();
+	public async findAll(name: string = ""): Promise<ProjectEntity[]> {
```

Comment:
Let's rename it to findAllByName, also name should be required
```suggestion
	public async findAllByName(name: string): Promise<ProjectEntity[]> {
```

Reply:
renamed